### DISCUSSION TITLE
Is there a way to lower Macbook integrated display saturation levels?

### DISCUSSION THREAD

**Discussion by axel9546** [MonitorControl/MonitorControl#1186] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

as title, Is there a way to lower Macbook integrated display saturation levels?

I found no option. 
If it would be possible with a feature add-on to the app, it would be phenomenal!

What do you think about?

**Comment by waydabber** [comment]
Saturation levels are determined by the ColorSync profile. You should think in reverse - if you choose a profile that is tuned for a lower gamut display than yours, saturation will increase and vice versa. You should choose a profile that is tuned for a wide-gamut display (one that can show more colors) than the built-in display, that will decrease the saturation.

Saturation cannot be changed by colortable manipulation - it would be possible to do it by using overlay filters, but altering the colorsync profile seems to be the better way to me.

**Reply by axel9546** [reply]
Thanks!
Which software do you reccomend to do a custom color profile that would match this scenario?
